gui/dist_view/distviewcompute.h

```diff
@@ -47,6 +47,10 @@ struct Bin {
 					   std::plus<multi_img::Value>());
 	}
 
+	inline void add(const cv::Vec3f &colorpixel) {
```

Comment:
please implement as a method with two arguments (pixel and color) that calls the other method (only pixel). so we don't have two add() calls in accumulate. same for sub.